### DISCUSSION TITLE
INAV with french language

### DISCUSSION THREAD

**Discussion by samsab1** [iNavFlight/inav-configurator#2291] [General]
hello
do you think that a french INAV version will be possible?
best regards from......france

**Comment by sensei-hacker** [comment]
You may have seen the translations people have submitted in Japanese, Ukrainian, etc.
There will be a France translation when it's submitted by someone from ... France.

**Comment by samsab1** [comment]
i would help if i know how to do that.
what's the way to try?

**Comment by sensei-hacker** [comment]
You copy locale/en/messages.json to locale/fr/message.json
Here are examples of the Japanese and Ukrainian. 

Then update another file or two as shown in these examples.

https://github.com/iNavFlight/inav-configurator/pull/2250/files

https://github.com/iNavFlight/inav-configurator/pull/2196
